tests/core/test_dataset.py

```diff
@@ -1167,10 +1252,10 @@ def test_partition(hexbeam):
 
 
 def test_explode(datasets):
-    for dataset in datasets:
-        out = dataset.explode()
-        assert out.n_cells == dataset.n_cells
-        assert out.n_points > dataset.n_points
+    for dset in datasets:
```

Comment:
Same with https://github.com/pyvista/pyvista/pull/7252#discussion_r1985810436

Reply:
Done in https://github.com/pyvista/pyvista/pull/7252/commits/8ee1db4903be48cb4226af2f67509f877dd7eae0